.gitignore

```
@@ -5,6 +5,10 @@
 /.pnp
 .pnp.js
 
+# JetBrains IDE files
+
+/.idea
```

Not needed, but maybe we should change it to:

```suggestion
.idea/
```